datayoga/blocks/expression.py

```diff
@@ -32,12 +32,58 @@ def search(self, data: Any) -> Any:
         """
         pass
 
+    def test(self, data: Any) -> bool:
+        """Test a where clause for an SQL statement
+
+        Args:
+            data (Any): Data
+
+        Returns:
+            boolean: True if matches, else False
+        """
+        pass
```

Comment:
why not implemented?

Reply:
removed. in the beginning I looped and ran 'test' but more efficient to just filter